### PR TITLE
Refactor pom, fix not working test, update scripts, add new script

### PR DESCRIPTION
**Pull request by ghost**
remove warning from mvn site
remove unnecessary pustefix dependency
refactor poms (reporting, descriptions, add plugin for run bash script)
update all scripts (we can run scripts from every path + error handling)
connect generate_reports.sh with mvn post-site (one command for generate all plugins)
add run_demo.sh for demonstrate project (git clone, run all plugins, run app gui)

### CODE REVIEW

File: client/src/main/java/com/blackship/battlesheep/fx/controllers/utils/PositionUtils.java [hunk 1]
```diff
@@ -6,13 +6,13 @@
  */
 public class PositionUtils {
 
-    public static int calculateFromTwoDimensionalPosition(int columns, int row, int column) {
+    public static int calculateFromTwoDimensionalPosition(int columns, int column, int row) {
         return row * columns + column;
     }
 
     public static int[] calculateFromOneDimensionalPosition(int position, int boardSize) {
```
Comment (mhbratek) [inline client/src/main/java/com/blackship/battlesheep/fx/controllers/utils/PositionUtils.java:13]: Why array?

File: common/src/test/java/com/blackship/battlesheep/communication/network/NetworkStreamsExceptionTest.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        givenNetworkStream = new NetworkStreams(givenSocket);
+    }
+
+    @Test
+    public void shouldReturnNoOutputStream() throws IOException {
+
+        //given
+
+        //when
+        when(givenSocket.getOutputStream()).thenThrow(new IOException());
```
Comment (mhbratek) [inline common/src/test/java/com/blackship/battlesheep/communication/network/NetworkStreamsExceptionTest.java:37]: :+1:

File: scripts/generate_reports.sh [hunk 3]
```diff
@@ -1,57 +1,140 @@
...
+do
+	check_install $var $PATH_TO_RUN
+done
+
+#run given plugins
+for var in "${@:2}"
+do
+	switch_input $var $PATH_TO_RUN
+done
+
```
Comment (mhbratek) [inline scripts/generate_reports.sh:122]: remove empty lines

**Review by mhbratek — APPROVED** [review]
:beers:

File: scripts/generate_reports.sh [hunk 4]
```diff
@@ -1,57 +1,140 @@
...
+
+
+
+
+
+
+
+
+
```
Comment (arturAR) [inline scripts/generate_reports.sh:131]: 👎

**Review by arturAR — APPROVED** [review]

**Pull request merged**